debugger.md

```diff
@@ -0,0 +1,83 @@
+ * **Magic step:** After the extension is installed, open a C# file. VSC might ask you to install "C#". Select to NOT install it! 
+ * In the background a process is triggered to get required bits. You can follow that process in the output window of VSC. Wait to finish that process.
+4. Install Mono 
+ * *Currently not required due to issue with command to create tasks.json*
+ * ~~To be able to auto-create a tasks.json for every project you'll currently need Mono. You could skip this step, but then you need to create the file manually.~~
+ * ~~Run the following commands in a terminal:~~
+ * ~~On Mac: brew install mono~~
+ * ~~On Linux: sudo apt-get install mono-complete~~
+
+###Once for each project
+The following steps have to executed for every project. 
+* Whenever you want to start debugging (e.g. by pressing the debugger-play button) a .NET Core app select '.NET Core"  for debug environment in the command pallette when VS code first asks for it. 
+* This will create a launch.json file. Enter the name of the executable in 'program' field (including the path). This could be something like *"${workspaceRoot}/bin/Debug/dnxcore50/osx.10.11-x64/HelloWorld"*. 
+* For attach scenarios modify the processName attribute of launch.json
+* If you want 'compile' support for F5 you need to create a tasks.json file . There are multiple ways to do this.
+  * You can get it here (\\vsdbgqa\x-plat\SampleApp\cli\.vscode\tasks.json) and put it manually next to your launch.json file 
+  * ~~Or you can open the command pallette in VSC (F1) and run the command *"Debugger: Add tasks.json"* (Make sure you have installed Mono to be able to do so, see above.
+  (If you don't have Mono installed you might get an error saying *Omnisharp not running*)~~ Unfortunately this is currently not working. We're investigating why.
+  * Or if you just want debugging, you can comment out 'preCompileTask' field​
+* **Important hint:**If you run into an error modify your project.json to reference "NETStandrad.Library":"1.0.0-rc3-*"
+  * Make sure the version number is correct. Currently dotnet new creates a project.json that references a version that's too old. You should also get a hint to modify this.
+  * Afterwards run dotnet restore.
+
+####Debugging Code compiled on another computer
+* The target app binaries must be built with Portable PDBs for symbol loading to work. 
+* If the target binary is built on Linux / OSX, dotnet CLI will produce portable pdbs by default so no action is necessary. 
+* If the target binary is built on windows, the pdb needs to be converted: How to convert Windows PDBs to Portable
+
+####More things to configure In launch.json
+#####Just My Code
+You can optionally configure justMyCode by setting it to “true”.
+>*“justMyCode”:true*
+
+#####Source File Map
+You can optionally configure a file by file mapping by providing map following this schema:
+
+>“sourceFileMap”:  {
+    “C:\\foo\bar.cs”:”/home/me/foo/bar.cs”
```

Comment:
I would suggest having your example be a directory rather than a source file. A source file works, but usually people would want to do a whole source tree. So:
"c:\\foo" : "/home/me/foo"